actors/evm/src/interpreter/system.rs

```diff
@@ -33,55 +32,51 @@ pub enum Call<'a> {
 /// that bridges the FVM world to EVM world
 pub struct System<'r, BS: Blockstore, RT: Runtime<BS>> {
     pub rt: &'r mut RT,
-    state: RefCell<Hamt<BS, U256, U256>>,
+    state: &'r mut Hamt<BS, U256, U256>,
```

Comment:
This looks like it's probably fine? We could go back to the RefCell, but it's more annoying.